src/main/java/net/minestom/arena/game/mob/MobArena.java

```diff
@@ -115,31 +114,12 @@ public MobArenaInstance() {
 
     private final Group group;
     private final Instance arenaInstance = new MobArenaInstance();
+    private final Set<Player> continued = new HashSet<>();
+    private final Map<Player, Integer> weaponTiers = new ConcurrentHashMap<>();
```

Comment:
Data maps should be replaced by a single `Map<Player, TagHandler>`

Reply:
Solved